db/migrate/20210910112644_create_stocks.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :stocks, id: false, primary_key: :code do |t|
       t.string :code
       t.string :name
-      t.float :current_price
+      # t.float :current_price
```

Comment:
If this `current_price` is already on the production database or on your teammate's database, then better to create a migration file removing this attribute.